src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -564,7 +564,8 @@ class ZStackAdapter extends Adapter {
                 );
                 const result = await response.start().promise;
                 if (result.payload.status !== ZnpCommandStatus.SUCCESS) {
-                    throw new Error(`LQI for '${networkAddress}' failed`);
+                    throw new Error(`LQI for '${networkAddress}' failed with status code '${
```

Comment:
I think this is a good improvement, but could you change the format to:

```js
LQI for '${networkAddress}' failed with error: '${ZnpCommandStatus[result.payload.status]}' (${result.payload.status})
```

Reply:
ok, done. I tried to add the same functionality to the ezsp adapter, yet the error code lookup did not work - so I skipped it for the ezsp adapter.